client/src/main.rs

```diff
@@ -31,7 +39,7 @@ async fn main() -> Result<()> {
 
     // Create the app and run it.
     let tick_rate = Duration::from_millis(35);
```

Comment:
This should be added to the `clap` args as well if we are working on this anyway.